integration-tests/contracts/ethereum_ocr2vrf_contracts.go

```diff
@@ -242,42 +291,28 @@ func (coordinator *EthereumVRFCoordinatorV3) Address() string {
 }
 
 func (coordinator *EthereumVRFCoordinatorV3) SetProducer(producerAddress string) error {
-	opts, err := coordinator.client.TransactionOpts(coordinator.client.GetDefaultWallet())
-	if err != nil {
-		return err
-	}
-	tx, err := coordinator.vrfCoordinatorV3.SetProducer(
-		opts,
+	_, err := coordinator.client.Decode(coordinator.vrfCoordinatorV3.SetProducer(
+		coordinator.client.NewTXOpts(),
 		common.HexToAddress(producerAddress),
-	)
-	if err != nil {
-		return err
-	}
-	return coordinator.client.ProcessTransaction(tx)
+	))
+	return err
 }
 
 func (coordinator *EthereumVRFCoordinatorV3) CreateSubscription() error {
-	opts, err := coordinator.client.TransactionOpts(coordinator.client.GetDefaultWallet())
-	if err != nil {
-		return err
-	}
-	tx, err := coordinator.vrfCoordinatorV3.CreateSubscription(
-		opts,
-	)
-	if err != nil {
-		return err
-	}
-	return coordinator.client.ProcessTransaction(tx)
+	_, err := coordinator.client.Decode(coordinator.vrfCoordinatorV3.CreateSubscription(
+		coordinator.client.NewTXOpts(),
+	))
+	return err
 }
 
 func (coordinator *EthereumVRFCoordinatorV3) FindSubscriptionID() (*big.Int, error) {
 	fopts := &bind.FilterOpts{}
-	owner := coordinator.client.GetDefaultWallet().Address()
+	owner := coordinator.client.Addresses[0]
```

Comment:
change to `client.MustGetRootKeyAddress()` pls; here and all the other occurrences

Reply:
Fixed in 340ddac38da63f9e789b06c47cee1fadaa1be23b